test/functional/driver/docker/scenarios/group_host_vars/molecule.yml

```diff
@@ -1,4 +1,12 @@
 ---
+ansible:
+  group_vars:
+    example:
+      - group_host_vars_group_molecule_yml: foo
+  host_vars:
+    group-vars-scenario:
+      - group_host_vars_host_molecule_yml: bar
+
```

Comment:
This doesn't appear to work.  Not sure why.  Do you have any idea @kireledan?

Doesn't seem that the host and group vars are getting created.